docs/discovery/provisioning.md

```diff
@@ -0,0 +1,456 @@
+# Provisioning Setup and Processing
+
+## Table of Contents
+
+- [Provisioning Setup and Processing](#provisioning-setup-and-processing)
+  - [Table of Contents](#table-of-contents)
```

Comment:
so, not sure you need this link right?  you have the header up top, and when you click this link, it stays on this page anyway?

Reply:
its added automatically